src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -5,6 +5,8 @@ import { AnyVenue, VenueTemplate } from "types/venues";
 
 import { FriendShipPage } from "pages/FriendShipPage";
 
+import { useProfileModalControls } from "hooks/useProfileModalControls";
+
```

Comment:
[nit] Import sort order. `pages/*` should really be the 'top-level' of the 'atomic design methodology' structure (pages, templates, organisms, molecules, atoms)

```suggestion
import { useProfileModalControls } from "hooks/useProfileModalControls";

import { FriendShipPage } from "pages/FriendShipPage";
```


Further reading:

- https://github.com/sparkletown/internal-sparkle-issues/issues/414
  - https://atomicdesign.bradfrost.com/chapter-2/